definitions/checks/maintenance_mode/check_consistency.rb

```diff
@@ -0,0 +1,65 @@
+module Checks::MaintenanceMode
+  class CheckConsistency < ForemanMaintain::Check
+    metadata do
+      label :check_maintenance_mode_consistency
+      description 'Check of maintenance-mode consistency'
+      confine do
+        feature(:downstream)
```

Comment:
Why these checks are downstream only?

Reply:
My bad. No need to limit check for :downstream here. I will remove it.